documentation/hiera.md

```diff
@@ -46,13 +46,11 @@ data set by a user overrides the default data set by the module’s author.
 ## Look up data from the command line
 
 You can use the `bolt lookup` command and `Invoke-BoltLookup` PowerShell cmdlet
-to look up data from the command line. The `lookup` command looks up data in the
-context of a target, allowing you to interpolate target facts and variables in
-your hierarchy.
+to look up Hiera data from the command line.
 
-> **Note:** The `bolt lookup` and `Invoke-BoltLookup` commands only look up data
-> using the `hierarchy` key in the Hiera configuration file. `plan_hierarchy`
-> is not supported from the command line.
+### Lookup up data from hierarchy key
```

Comment:
We don't mention the hierarchy key in this section outside of the section title. We've told the reader to learn more about Hiera and linked off to the Hiera docs, but I think we should add some references to where the hierarchy key is. I'll look for some opportunities do that below.

This command line section is a little difficult to parse right now because we've added a section on the `plan_hierarchy` key, but that key is only explained later in the plans section. The `plan_hierarchy` key was mentioned in a note that we're removing here, and I think I just missed this last time I reviewed. I think we should:
- Move the `plan_hierarchy` lookup section under "Look up data in plans" (It can probably just go right at the end).
- Add a sentence to the end of the "Look up data from the command line" section that says something like: "To look up data outside an apply block in a Bolt plan, see..."
- Remove this section title (I added a piece above that should send users in the right direction if they're 
   looking for info on lookups outside apply blocks

Reply:
Mmk, I really don't like the titles I gave the subheadings of `Look up data on the command line`, but I'm not sure if it's better to continue the theme of 'inside apply blocks / outside apply blocks' or to use more correct language around target context vs. plan context (or to even just say "from hierarchy" and "from plan_hierarchy")